partiql-ast/src/main/java/org/partiql/ast/v1/expr/SessionAttribute.java

```diff
@@ -49,4 +49,12 @@ public static SessionAttribute[] values() {
             CURRENT_DATE()
         };
     }
+
+    public String name() {
```

Comment:
Something else I noticed was missing from the AST enum classes, was the `name` property (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-enum/name.html). Added since it was used for one of the ast visitors. Perhaps we should add it for all of the AST enums?

Reply:
TODO: add `name` to `enum` methods to Java interface.

Reply:
Added to the `AstEnum` interface in https://github.com/partiql/partiql-lang-kotlin/pull/1630.